### ISSUE TITLE
Buffer size not sufficent

### ISSUE THREAD

**Issue by bodieacb** [morkt/GARbro#135]
I get this issue on many images and they don't show up. I wasn't getting this issue earlier today. I also get issues relating to an index like, "Index was out of range. Must be non-negative and less than the size of the collection. Parameter name: index" and "Index was outside the bounds of the array." What does it mean?

**Comment by morkt** [comment]
it means your game is not supported.

**Comment by bodieacb** [comment]
But it was working fine like an hour or two before I made the post.

**Comment by morkt** [comment]
then you did something in those hour or two that now prevents application from working. but seriously, can't you share some details, like game title, garbro version, what file you're trying to open, what exactly happening. basic stuff for any issue reporting.

**Comment by morkt** [comment]
closing as insubstantial, feel free to re-open or open a new issue, but make sure to provide more details.

**Comment by fighterup** [comment]
I'm having this same issue with a game called Kemo Coliseum. I'm using garbro version 1.5.35. I try and open audio or image files and they don't open. they get the same errors.

**Comment by fighterup** [comment]
I'm having this same issue with a game called Kemo Coliseum. I'm using garbro version 1.5.35. I try and open audio or image files and they don't open. they get the same errors. it seems to be a similar issue to https://github.com/morkt/GARbro/issues/20. I have both of the things listed as installed in those solutions and the dll files don't have the option to unblock them.

**Comment by morkt** [comment]
it can't be similar to #20 because then it would be resolved after unblocking. it's something completely different. anyway, i can't reproduce it, so there's nothing i can do about it.
